### PR TITLE
[13.0][MIG] stock_picking_late_activity: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT27179

Normal migration.

### CODE REVIEW

File: stock_picking_late_activity/tests/test_stock_picking_late_activity.py [hunk 1]
```diff
@@ -63,7 +64,6 @@ def create_picking(cls, partner, picking_type, product, scheduled_date):
             ),
             view="stock.view_picking_form",
         )
-        picking_form.company_id = cls.env.user.company_id
```
Comment (pedrobaeza) [inline stock_picking_late_activity/tests/test_stock_picking_late_activity.py]:
```suggestion
        picking_form.company_id = cls.env.company
```

**Review by chienandalu — APPROVED** [review]
Tested :+1:

File: stock_picking_late_activity/tests/test_stock_picking_late_activity.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+from datetime import datetime, timedelta
+
+from odoo.tests import Form, common, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestStockPickingLateActivity(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestStockPickingLateActivity, cls).setUpClass()
```
Comment (joao-p-marques) [inline stock_picking_late_activity/tests/test_stock_picking_late_activity.py:13]:
```suggestion
        super().setUpClass()
```

**Review by joao-p-marques — APPROVED** [review]
Code and functional review :+1: 
Just a minor comment.

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**